templates/app/contentful-ssg.config.ts

```diff
@@ -5,7 +5,15 @@ const config: Config = {
   richTextRenderer: false,
   preview: process.env.CONTENTFUL_USE_PREVIEW === 'true',
   plugins: [
-    '@jungvonmatt/cssg-plugin-hugo',
+    [
+      '@jungvonmatt/cssg-plugin-hugo',
+      {
+        menuRootTypes: ['t-*'],
```

Comment:
have you checked if globs work here?
Otherwise we could also try to read the hugo config to grab the page types from there and allow`page` & `t-*` as fallback so we keep backwards compatible
```
menuRootTypes: ['t-*','page']
```

Reply:
Just checked the implementation at `cssg-plugin-hugo`. 
menuRootTypes option couldn't handle globs at the moment so we need to use `t-page` here

Reply:
Done